settings/config.settings.php

```diff
@@ -20,21 +20,27 @@
  * $settings['config_sync_directory'] = $dir . "/config/$site_dir" in
  * settings.php, and we will not overwrite it.
  */
-
+// phpcs:ignore
+if (!isset($drs_override_config_directories)) {
+  $drs_override_config_directories = TRUE;
+}
 
 /**
  * Site path.
  *
  * @var string $site_path
  * This is always set and exposed by the Drupal Kernel.
  */
- // phpcs:ignore
- $site_name = EnvironmentDetector::getSiteName($site_path);
 // phpcs:ignore
-// Config sync settings.
-$settings['config_sync_directory'] = "../config/" . $site_name;
-// Site Studio sync settings.
-$settings['site_studio_sync'] = "../sitestudio/" . $site_name;
+$site_name = EnvironmentDetector::getSiteName($site_path);
+
+// phpcs:ignore
+if ($drs_override_config_directories) {
+  // Config sync settings.
+  $settings['config_sync_directory'] = "../config/" . $site_name;
```

Comment:
@apathak18 - This will follow the similar logic what you have done at line 24-26, also keep the variable separate in respective if condition block.